inspire_json_merger/comparators.py

```diff
@@ -85,6 +85,28 @@ class Ret(PrimaryKeyComparator):
     return Ret
 
 
+class NoMatch(object):
+    """Ensure the result of a normalization function is not a match."""
+    def __equals__(self, other):
+        return False
+
+
+class SubdictNormalizer(object):
+    """Normalizer that only looks at the keys passed as argument.
+    
+    If any required key is not present, the element won't match.
+    """
+    def __init__(self, keys):
+        self.keys = set(keys)
+
+    def __call__(self, field):
+        if not self.keys.issubset(field.keys()):
+            return NoMatch()
+
+        return {k: field[k] for k in self.keys}
```

Comment:
I did something simple here. Ideally we would support something similar to the `PrimaryKeyComparator`, but that's a bit more complex to implement.